modules/etcd-creating-automated-backups.adoc

```diff
@@ -0,0 +1,355 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc
+
+:_content-type: PROCEDURE
+[id="creating-automated-etcd-backups_{context}"]
+= Creating automated etcd backups
+
+The automated backup feature for etcd supports both recurring and single backups. Recurring backups create a cron job that starts a single backup each time it triggers.
+
+:FeatureName: Automating etcd backups
+include::snippets/technology-preview.adoc[]
+
+[id="enabling-automated-etcd-backups_{context}"]
+== Enabling automated etcd backups
+
+Follow these steps to enable the automated backups feature for etcd.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a `FeatureGate` custom resource (CR) file named `etcd-enable-automated-backups.yaml` with the following contents:
```

Comment:
In other places in this file, you use the phrase "with content such as the following example:" to lead into a code example. For consistency, I'd suggest using the same phrase throughout the file, unless in certain cases (like this one) you really mean that the user should use the exact contents that are provided.

Reply:
Yes, in this case, they are supposed to copy the literal contents here. I use "such as" to indicate that changes may be required to the example content.